tool/tctl/common/bots_command.go

```diff
@@ -238,9 +283,156 @@ func (c *BotsCommand) AddBot(ctx context.Context, client auth.ClientI) error {
 	})
 }
 
+// AddBot adds a new certificate renewal bot to the cluster.
+func (c *BotsCommand) AddBot(ctx context.Context, client auth.ClientI) error {
+	// Jankily call the endpoint invalidly. This lets us version check and use
+	// the legacy version of this CLI tool if we are talking to an older
+	// server.
+	{
```

Comment:
Perhaps deserving of a `// DELETE in X.Y`

Reply:
e472f48327